client/my-sites/theme/main.jsx

```diff
@@ -16,6 +16,7 @@ import Main from 'components/main';
 import HeaderCake from 'components/header-cake';
 import SectionHeader from 'components/section-header';
 import ThemeDownloadCard from './theme-download-card';
+import ThemeRelatedThemesCard from './theme-related-themes-card';
```

Comment:
Well it was named RelatedThemesCard before I have stumbled upon naming requirements for sub-components in the documentation:
>  Classes (both in components and sub-components) are always prefixed to the `component` name.
You are right a bit lengthy this way... 
Is there a tiebreaker rule like: in the end use common sense?  That can be applied here to shorten the name?